Sources/PlatformVersion.swift

```diff
@@ -416,7 +422,9 @@ extension visionOSVersion {
         if #available(visionOS 27, *) {
             return .past
         }
-        if #available(visionOS 26, *) {
+        // Apps built before the visionOS 26 SDK get "3.0" as the system version from ProcessInfo.
+        // Once built with the visionOS 26 SDK, the version then becomes "26.0".
+        if #available(visionOS 3, *) {
```

Comment:
I don't know this for sure, I'm just assuming based on what the iOS SDK does. In any case it's safe to do as visionOS 3 will never actually exist.